single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/styles/iidm/TopologicalStyleProvider.java

```diff
@@ -176,19 +183,18 @@ private boolean isMultiTerminalInternalNode(Node node) {
     }
 
     @Override
-    public Optional<String> getVoltageLevelNodeStyle(VoltageLevelInfos voltageLevelInfos, Node node) {
+    public List<String> getVoltageLevelNodeStyle(VoltageLevelInfos voltageLevelInfos, Node node) {
         if (node.getType() == NodeType.SWITCH && ((SwitchNode) node).isOpen()) {
-            return Optional.of(StyleClassConstants.DISCONNECTED_STYLE_CLASS);
+            return List.of(StyleClassConstants.DISCONNECTED_STYLE_CLASS);
         }
-        String style = Optional.ofNullable(voltageLevelInfos)
+        return Optional.ofNullable(voltageLevelInfos)
                 .flatMap(vli -> baseVoltagesConfig.getBaseVoltageName(vli.getNominalVoltage(), BASE_VOLTAGE_PROFILE))
-                .flatMap(baseVoltageName -> getNodeTopologicalStyle(StyleClassConstants.STYLE_PREFIX + baseVoltageName, voltageLevelInfos.getId(), node))
-                .orElse(StyleClassConstants.DISCONNECTED_STYLE_CLASS);
-        return Optional.of(style);
+                .map(baseVoltageName -> getNodeTopologicalStyle(StyleClassConstants.STYLE_PREFIX + baseVoltageName, voltageLevelInfos.getId(), node))
```

Comment:
Just realized it was quite strange to have the "sld-" prefix in the map, so could you pass the baseVoltageName here instead of what was done previously?
```suggestion
                .map(baseVoltageName -> getNodeTopologicalStyle(baseVoltageName, voltageLevelInfos.getId(), node))
```